package.json

```diff
@@ -1,3 +1,25 @@
 {
-    "type": "module"
-}
\ No newline at end of file
+  "name": "pocket-archiver",
```

Comment:
Adding the following fields to your package.json helps define your project properties and makes it easier to share with others.

Running `npm init -y` scaffolds the package.json.

You should also add dependencies here. For example, `npm install --save node-fetch`.